admin/src/js/services/location.js

```diff
@@ -5,7 +5,7 @@ angular.module('inboxServices').factory('Location',
     'ngInject';
 
     const location = $window.location;
-    const dbName = 'medic';
+    const dbName = $window.localStorage.getItem('dbName') || 'medic';
```

Comment:
For db kind of things I prefer to be more restrictive, many things can happen with a wrong db:
```
const isTestEnv = $window.localStorage.getItem('isTestEnv');
const dbName = isTestEnv ? 'medic-test' : 'medic';
```